components/position-summary.js

```diff
@@ -0,0 +1,57 @@
+import { FaListUl } from "react-icons/fa";
+import { BsCurrencyDollar } from "react-icons/bs";
+import { FaUserAlt } from "react-icons/fa";
+import { FaLightbulb } from "react-icons/fa";
+import { FaLayerGroup } from "react-icons/fa";
+
+export default function PositionSummary() {
+  return (
+    <div className=" bg-light h-auto w-80">
+      <div className="ml-5 grid grid-cols-2 w-54 mt-5 mb-6">
+        <div className="">
+          <BsCurrencyDollar className="text-4xl text-accent" />
+        </div>
+        <div className="-ml-20">
+          <h1 className="text-xl text-secondary font-bold">offered salary</h1>
+          <h1 className="text-base">$45K-$180K</h1>
```

Comment:
data shouldn't be hard coded. Instead, a summary object needs to be present at the top:

```js
const summary = {
  salaryMin: 45000,
  salaryMax: 180000,
  currency: { name: 'United States Dollars', symbolShort: '$', symbolLong: "USD" },
  gender: 'any',
  experience: "2+ years",
  workLevel: "senior",
  category: "Web Development"
}
```

Reply:
then render these data as required